### DISCUSSION TITLE
Shelly Plus Add-on  experience

### DISCUSSION THREAD

**Discussion by HB787** [arendst/Tasmota#19879] [Q&A]
Hi team,
I am trying to use the Shelly Plus Add-On in combination with a Shelly Plus 2PM. 
The DHT22 is working based on pins:
GPIO0 DHT11_o
GPIO1 AM2301

Now my challenge is to get two DS18B20 Temp sensors to work with Tasmota.
I also tried to let configure a HC-SR501 PIR without success.
Within the Shelly firmware it all worked.

Thanks in advance for any help.
Harry

**Comment by sfromis** [comment]
Already being discussed in https://github.com/arendst/Tasmota/discussions/18885
